src/test/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexp/InputRegexpCheckB4.java

```diff
@@ -0,0 +1,3 @@
+
+package com.puppycrawl.tools.checkstyle.checks.regexp.regexp; // violation
```

Comment:
Can we show violation message?
If line is long, use `violation above`

Reply:
done I have changed  config and file to not have  only package